lib/convict.js

```diff
@@ -439,7 +445,7 @@ var convict = function convict(def) {
   // If the definition is a string,
   // treat it as an external schema file.
   if (typeof def === 'string') {
-    rv._def = cjson.load(def);
+    rv._def = loadJSON(def);
```

Comment:
Here it's more difficult to do an async file read, but if you can spend some time trying to figure it out that would be good. If it's too difficult please add a TODO specifying our will to get rid of the sync call.